specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -556,577 +637,1126 @@
+    },
+    "PrecipitationType": {
+      "type": "string",
+      "description": "Specifies the type of precipitation (\"Rain\" \"Snow\" \"Ice\" or \"Mix\"). If dbz is zero, precipitationType is not present in the response.",
+      "enum": [
+        "Rain",
+        "Snow",
+        "Ice",
+        "Mix"
+      ],
+      "x-ms-enum": {
+        "name": "PrecipitationType",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Rain",
+            "description": "Rain"
+          },
+          {
+            "value": "Snow",
+            "description": "Snow"
+          },
+          {
+            "value": "Ice",
+            "description": "Ice"
+          },
+          {
+            "value": "Mix",
+            "description": "Mix"
+          }
+        ]
+      }
+    },
+    "WeatherUnit": {
+      "x-ms-client-name": "WeatherValue",
+      "description": "Specific value of a given unit related to weather.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "number",
+          "format": "float",
+          "description": "Rounded value."
+        },
+        "unit": {
+          "x-ms-client-name": "unitLabel",
+          "type": "string",
+          "description": "Type of unit for the returned value."
+        },
+        "unitType": {
+          "$ref": "#/definitions/UnitType"
+        }
+      }
+    },
+    "WindDetails": {
+      "type": "object",
+      "description": "Wind details being returned including speed and direction.",
+      "properties": {
+        "direction": {
+          "$ref": "#/definitions/WindDirection",
+          "description": "Wind direction"
+        },
+        "speed": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "Speed of the wind in specified unit."
+        }
+      }
+    },
+    "WindDirection": {
+      "type": "object",
+      "properties": {
+        "degrees": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Wind direction in Azimuth degrees,  starting at true North and continuing in clockwise direction. North is 0 degrees, east is 90 degrees, south is 180 degrees, west is 270 degrees. Possible values 0-359."
+        },
+        "localizedDescription": {
+          "x-ms-client-name": "description",
+          "type": "string",
+          "description": "Direction abbreviation in the specified language."
+        }
+      },
+      "description": "Wind direction"
+    },
+    "WindSpeed": {
+      "type": "object",
+      "description": "Speed of wind in specified unit.",
+      "properties": {
+        "value": {
+          "type": "number",
+          "format": "float",
+          "description": "Rounded value of the speed."
+        },
+        "unit": {
+          "type": "string",
+          "description": "Type of unit for the speed value."
+        },
+        "unitType": {
+          "$ref": "#/definitions/UnitType"
+        }
+      }
+    },
```

Comment:
This is not referenced by anything and should be removed.

Reply:
WindSpeed is used in windRadiiSummary, and windRadiiSummary is used in StormForecast and StormLocation.

Reply:
It isn't. They use WeatherUnit. Nothing in the Weather Swagger references this definition (which is just  duplicate of WeatherUnit anyhow).

Reply:
Thanks for the remind! I just removed it.